Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM node:16-alpine3.15 as js-builder
+FROM node:16-alpine3.15 AS js-builder
 
-ENV NODE_OPTIONS=--max_old_space_size=8000
+ENV NODE_OPTIONS --max_old_space_size=8000
```

Comment:
Actually, the warning said that you should use ENV=VALUE instead of ENV VALUE. Also https://docs.docker.com/reference/dockerfile/#env  says that ENV VALUE is discouraged.
Maybe the warning could be helped by putting quotation marks around the value?

Reply:
Oh, you're right. In commit be26e09e14e04188d7b4a5755b30713b55a2fb1b.